### DISCUSSION TITLE
Inclusion of ... argument in Spectre functions that call other functions

### DISCUSSION THREAD

**Discussion by hrj21** [ImmuneDynamics/Spectre#101] [! News]
Hi lovely people! Me again :P

There are some functions that Spectre calls from other packages (e.g. cytonorm, flowsom etc) where the functions from those packages have additional arguments I would sometimes like to use, but don't have access to when interfacing with these functions via Spectre.

For example I would to provide the `plot` and `normParams = list(nQ = 101, goal = "mean", limits = c(0, 10))` arguments to the `CytoNorm.train()` function that is called by `train.cytonorm()`. I know it would be cumbersome to add each argument as a named argument to each Spectre function, but could this be done by simply adding the `...` argument that lets us pass further arguments to internal function calls?

**Comment by SamGG** [comment]
Just for referencing the valuable feedback from Sofie https://github.com/saeyslab/CytoNorm/issues/28 at the interesting assay of hrj21

**Comment by tomashhurst** [comment]
Good call! We'll add this to the list.
